src/foam/core/types.js

```diff
@@ -741,6 +741,15 @@ foam.CLASS({
       value: function(value, cloneMap, opt_X) {
         cloneMap[this.name] = value && value.clone ? value.clone(opt_X) : value;
       }
+    },
+    {
+      name: 'copyTransient',
+      class: 'Boolean',
+      value: false,
```

Comment:
When/why would we want to make this configurable?

Reply:
Originally I thought it was needed because the client build stopped working, but then making the change to put this here fixed the issue by itself. I left it here in case it turns out to cause a problem somewhere else (since it's a regression if any code expects the object to hold a reference to an FObject in a property).

Should I remove this?